CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+[SublimeText][sublimetext-download], or [Atom][atom-download]) to make changes to the files.
+
+Once you have made the necessary edits, you can commit these edits by running the 
+following commands. Here, we assume that I have added a new file called "article.md" 
+to the repository:
+
+```
+$ git add article.md 
+$ git commit -m "adding my new article"
+```
+
+#### 5. Pushing your changes to GitHub
+
+Once that runs, you can "push" these changes to GitHub.
+
+The first time you push your changes, you will need to specify your upstream branch 
+by running the following command:
+
+```
+git push -set-upstream my-new-constribution
+```
+> **Note**
+> Any further push commands will only need to be ```git push``.
+
+The output of your first push command always includes a link you can use to open a 
+pull request. Click this link to open a pull request against the main 
+branch in the conda-dot-org repository. On the pull request creation form, always
+be sure that the "base" branch is the "main" branch for conda-dot-org.
+
+That completes the entire workflow for submitting new changes. After you submit your pull
+request, we will try our best to review it in a timely manner. If you need to make any more
+edits or changes based on these reviews, repeat steps four and five (but using ```git push```
+only to push any more changes).
+
+## Setting up a local version of the site
+
+The above steps are great for adding your changes to git and submitting them for review,
+but what if you want to preview exactly what your changes will look like on the website?
+The answer to this involves running a local version of the website that automatically 
+updates as you change and add files on your computer. Below, we will walk through the
+steps necessary to do just that.
+
+### Prerequisites
+
+- npm: [installation link][installing-npm]
+
+NPM (Node Package Manager) is a program used for installing Javascript dependencies on your
+computer. To follow this guide, it is not necessary that you know Javascript, but you will 
+need to be comfortable with running a couple different commands in your command line 
+interface (CLI) that will install these dependencies and then run the server locally.
+
+### Installing NPM dependencies
+
+Provided that you have already forked and cloned the repository (see above), the first step 
+is installing all the necessary dependencies.
+
+If you aren't in the conda-dot-org repository folder, use the `cd` command to change your
+directory. Consider the following directory structure:
+
+```
+├── Users
+│   ├── jdoe *You are here
+│   │   ├── repos
+│   │   │   ├── conda-dot-org
+```
+
+When you first open a CLI, it usually opens in your home directory. You
+can use `cd`, followed by a file path, to properly change your directory.
+
+```
+cd repos/conda-dot-org
+```
+
+Then, use the following command to install your dependencies:
+
+```
+npm install 
+```
+
```

Comment:
Do want to let them know that they may see warnings, and what those warnings might look like?  For example

This may result in several warnings being issued. For example:

```
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility

added 1091 packages, and audited 1092 packages in 26s

199 packages are looking for funding
  run `npm fund` for details

24 vulnerabilities (8 moderate, 16 high)

To address issues that do not require attention, run:
  npm audit fix

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
```
These errors can be ignored.

Reply:
So, the doc talks about the error log at the very end, but these are specifically the warnings you get when you run `npm install` and not `npm run start`, correct?

Reply:
I pushed all changes except for ones concerning this comment for now.

Reply:
Yes, these are the ones I got when I ran `npm install`. But it is really hard to know what types of warnings will be returned in a week or a month.  No idea how to anticipate what errors are "significant.